### PR TITLE
Update zh-HANT.coco.coffee

### PR DESCRIPTION
**Pull request by chengchiehhung**
Translated some tags.

### CODE REVIEW

File: app/locale/zh-HANT.coco.coffee [hunk 1]
```diff
@@ -242,11 +242,11 @@ module.exports = nativeDescription: "繁體中文", englishDescription: "Chinese
     open_source: "開源 (GitHub)"
     support: "取得幫助"
     faqs: "FAQs常見問題"
-#    copyright_prefix: "Copyright"
-#    copyright_suffix: "All Rights Reserved."
+    copyright_prefix: "著作權"
+    opyright_suffix: "保留所有權利"
```
Comment (nwinter) [inline app/locale/zh-HANT.coco.coffee:246]:
```suggestion
    copyright_suffix: "保留所有權利"
```

File: app/locale/zh-HANT.coco.coffee [hunk 2]
```diff
@@ -242,11 +242,11 @@ module.exports = nativeDescription: "繁體中文", englishDescription: "Chinese
...
     help_pref: "需要協助嗎? 寫封Email給我們"
     help_suff: "然後我們會與您接觸!"
-#    resource_hub: "Resource Hub"
+    esource_hub: "資源區域"
```
Comment (nwinter) [inline app/locale/zh-HANT.coco.coffee:249]:
```suggestion
    resource_hub: "資源區域"
```

**Review by nwinter — CHANGES_REQUESTED** [review]
Thanks! Lost the initial character on `copyright_suffix` and `resource_hub`; can you add those back?

**Comment by chengchiehhung** [comment]
Thanks for pointing out. Fixed it.

**Review by nwinter — APPROVED** [review]

**Pull request merged**